aws/table_aws_wellarchitected_lens.go

```diff
@@ -0,0 +1,243 @@
+package aws
+
+import (
+	"context"
+	"errors"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected"
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected/types"
+	"github.com/aws/smithy-go"
+
+	wellarchitectedv1 "github.com/aws/aws-sdk-go/service/wellarchitected"
+
+	"github.com/turbot/go-kit/helpers"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsWellArchitectedLens(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_wellarchitected_lens",
+		Description: "AWS Well-Architected Lens",
+		List: &plugin.ListConfig{
```

Comment:
@ParthaI We need to add a `GetConfig` for this table, in particular because it lets us get information about a lens for a particular version by passing in the `LensVersion` param. The `GetConfig` should take in `lens_alias` as a required qual and `lens_version` as an optional qual with exact cache match.

There are some properties only available in the `ListLenses` API call, so those columns may need a hydrate with the list function (I believe this is how we do it)